unit-tests/syncer/sw.py

```diff
@@ -61,6 +64,46 @@ def init():
     #
     global syncer
```

Comment:
Where is this "syncer" defined.
I see you use it but no global "syncer" is defined, am I missing it?

Reply:
+ You set a rs.syncer on "init()" and overrun it on on "playback".
why?
I think this file needs more comments.
It's hard to understand how to use it.
what is the purpose of this helper file, how can we use it? stuff like that.

Reply:
Agree, it just grew as I added to it.
Added comments and even an exception.